en/components/tabs.md

```diff
@@ -38,15 +38,50 @@ import { IgxTabsModule } from 'igniteui-angular';
 export class AppModule {}
 ```
 
-We set the names of tabs using the [`label`]({environment:angularApiUrl}/classes/igxtabsgroupcomponent.html#label) attribute. Any content provided between the opening and closing `igx-tabs-group` tags will appear in the respective tab group as content.
+We set the tabs header by providing a content between [`igx-tab-header`]({environment:angularApiUrl}/classes/igxtabheader.html}). To set the tabs name we simply add a span with [`igxTabHeaderLabel`]({environment:angularApiUrl}/classes/igxtabheaderlabeldirective.html) directive, between the `igx-tab-header` tags. Any content that will appear as a tab item's content should be added between [`igx-tab-content`]({environment:angularApiUrl}/classes/igxtabcontent.html}) tags.
```

Comment:
'...by providing content between [`igx-tab-header`]({environment:angularApiUrl}/classes/igxtabheader.html}) tags. To set the tab's name we simply add a span with [`igxTabHeaderLabel`]({environment:angularApiUrl}/classes/igxtabheaderlabeldirective.html) directive.

Reply:
Fixed

Reply:
The first sentence "We set the tabs header by **providing a content between [`igx-tab-header`]**" -> "**providing content to [`igx-tab-header`]**" sounds better.

Reply:
Changed